### PR TITLE
[Public Cloud] Add installation of Google SDK

### PR DESCRIPTION
**Pull request by jlausuch**
- Related ticket: https://progress.opensuse.org/issues/38741
- Verification run: http://fromm.arch.suse.de/tests/1765#

### CODE REVIEW

File: tests/publiccloud/prepare_tools.pm [hunk 1]
```diff
@@ -71,6 +71,14 @@ sub run {
...
     zypper_call('addrepo --name "Azure CLI" --check https://packages.microsoft.com/yumrepos/azure-cli azure-cli');
     zypper_call('install --from azure-cli -y azure-cli');
+
+    # Install Google Cloud SDK
+    assert_script_run("touch ~/.bashrc");
+    assert_script_run("wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-220.0.0-linux-x86_64.tar.gz");
+    assert_script_run("tar zxvf google-cloud-sdk-220.0.0-linux-x86_64.tar.gz google-cloud-sdk");
+    assert_script_run("cd google-cloud-sdk/ && ./install.sh -q");
+    assert_script_run("cp /root/google-cloud-sdk/completion.bash.inc ~/.bashrc");
+    assert_script_run("cp /root/google-cloud-sdk/path.bash.inc ~/.bashrc");
```
Comment (cfconrad) [inline tests/publiccloud/prepare_tools.pm]: doesn't this overwrite the `.bashrc` and previous cp is useless?

Comment (jlausuch) [reply tests/publiccloud/prepare_tools.pm]: yes, just realized... fixing it now

**Comment by jlausuch** [comment]
@asmorodskyi  if you agree, I will merge

File: tests/publiccloud/prepare_tools.pm [hunk 2]
```diff
@@ -71,6 +71,14 @@ sub run {
     assert_script_run('sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc');
     zypper_call('addrepo --name "Azure CLI" --check https://packages.microsoft.com/yumrepos/azure-cli azure-cli');
     zypper_call('install --from azure-cli -y azure-cli');
+
+    # Install Google Cloud SDK
+    assert_script_run("touch ~/.bashrc");
+    assert_script_run("wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-220.0.0-linux-x86_64.tar.gz");
+    assert_script_run("tar zxf google-cloud-sdk-220.0.0-linux-x86_64.tar.gz google-cloud-sdk");
+    assert_script_run("cd google-cloud-sdk/ && ./install.sh -q");
+    assert_script_run("echo . /root/google-cloud-sdk/completion.bash.inc >> ~/.bashrc");
```
Comment (asmorodskyi) [inline tests/publiccloud/prepare_tools.pm]: this will work even without line 76 , maybe better drop line 76 ?

Comment (jlausuch) [reply tests/publiccloud/prepare_tools.pm]: right, ok

**Comment by asmorodskyi** [comment]
LGTM but let *me* to merge it :) let's keep merging own PRs only for special occasions :) 

P.S. will wait for travis and will do it if this guy will not mind

**Comment by okurz** [comment]
@asmorodskyi I do not see a problem with anyone merging his own PR. *My* rule is: There should be peer-review. But this could be fulfilled by me looking over your shoulder and stating "ok" and then you press the merge button on your own PR because there *was* code-review ;)

**Pull request merged**